src/main/avro/com.snowplowanalytics.sauna.responders/SlackConfig/avro/1-0-0.avsc

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
Also added to iglu-central by https://github.com/snowplow/iglu-central/pull/498.